rosbag2_performance/rosbag2_performance_benchmarking/include/rosbag2_performance_benchmarking/config_utils.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020-2021, Robotec.ai sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROSBAG2_PERFORMANCE_BENCHMARKING__CONFIG_UTILS_HPP_
+#define ROSBAG2_PERFORMANCE_BENCHMARKING__CONFIG_UTILS_HPP_
+
+#include <string>
+#include <vector>
+
+#include "rclcpp/node.hpp"
+#include "rclcpp/qos.hpp"
+
+#include "rosbag2_performance_benchmarking/bag_config.hpp"
+#include "rosbag2_performance_benchmarking/producer_config.hpp"
+#include "rosbag2_performance_benchmarking/publisher_group_config.hpp"
+
+namespace config_utils
+{
+
+void load_qos_configuration(
```

Comment:
do all these functions need to be declared `inline`? I assume that right now it builds because this is only included in one place

Reply:
code for utils has been moved to source files. Some comments on functions in headers were added. This also helps with readability.